src/main/java/org/opentripplanner/routing/core/SimpleIntersectionTraversalCostModel.java

```diff
@@ -30,20 +30,32 @@
     /** Expected time it takes to turn left without a stop light. */
     private Double expectedLeftNoLightTimeSec = 8.0;
 
+    private double cyclingRightTurnMultiplier = 3.5;
+
+    /** Since doing a left turn on a bike is quite dangerous we add a cost for it**/
+    private double cyclingLeftTurnMultiplier = cyclingRightTurnMultiplier * 2;
+
     @Override
     public double computeTraversalCost(IntersectionVertex v, StreetEdge from, StreetEdge to, TraverseMode mode,
-                                       RoutingRequest options, float fromSpeed, float toSpeed) {
+                                       RoutingRequest request, float fromSpeed, float toSpeed) {
 
         // If the vertex is free-flowing then (by definition) there is no cost to traverse it.
         if (v.inferredFreeFlowing()) {
             return 0;
         }
 
-        // Non-driving cases are much simpler. Handled generically in the base class.
-        if (!mode.isDriving()) {
-            return computeNonDrivingTraversalCost(v, from, to, fromSpeed, toSpeed);
+        if (mode.isDriving()) {
+            return computeDrivingTraversalCost(v, from, to, request);
+        }
+        else if(mode.isCycling()) {
```

Comment:
An alternative would be to leave SimpleIntersectionTraversalCostModel as is and have our changes in a new CostModel. That way, we'd minimize conflicts with others and allow people to switch the implementation.